playwright.config.ts

```diff
@@ -0,0 +1,90 @@
+import { defineConfig, devices } from '@playwright/test';
```

Comment:
remove this file,execute E2E at test directory
<img width="669" alt="image" src="https://user-images.githubusercontent.com/15225835/226526105-64090450-b02b-42a9-9841-be8dcc95d6fb.png">